ipatests/test_integration/test_external_ca.py

```diff
@@ -411,3 +411,93 @@ def test_master_install_ca2(self):
             'certutil', '-L', '-d', paths.PKI_TOMCAT_ALIAS_DIR,
             '-n', cert_nick])
         assert "CN=RootCA2" in result.stdout_text
+
+
+def _step1_profile(master, s):
+    return install_server_external_ca_step1(
+        master,
+        extra_args=['--external-ca-type=ms-cs', f'--external-ca-profile={s}'],
+        raiseonerr=False,
+    )
+
+
+def _test_invalid_profile(master, profile):
+    result = _step1_profile(master, profile)
+    assert result.returncode != 0
+    assert '--external-ca-profile' in result.stderr_text
```

Comment:
Wouldn't it be better to expect certain output? In at least the case of "NotAnOid:1" it returns a non-localized error message so comparing all or part of that would be more complete.

Reply:
It tests that the output refers to the `--external-ca-profile` option.  This is good enough IMO.  Otherwise tests become more brittle in the face of trivial string changes or when localisation is implemented.